index.d.ts

```diff
@@ -45,6 +45,13 @@ export declare interface Tracer extends opentracing.Tracer {
    */
   init(options?: TracerOptions): this;
 
+  /**
+   * Sets the URL for the trace agent. This should only be called _after_
+   * init() is called, only in cases where the URL needs to be set after
+   * initialization.
+   */
+  setUrl(url: string): void;
```

Comment:
Small nit: returning the tracer would allow chaining.

Reply:
done